dags/main_summary_backfill.py

```diff
@@ -18,8 +18,8 @@
 
 t0 = EMRSparkOperator(task_id="main_summary_backfill",
                       job_name="Main Summary Backfill",
-                      execution_timeout=timedelta(hours=10),
-                      instance_count=20,
+                      execution_timeout=timedelta(hours=24),
```

Comment:
Is this enough for 7 days? Seems like we may want to up this to 48 (or even 60?) until we know what the actual runtime is